docker-compose.madoc.yml

```diff
@@ -0,0 +1,32 @@
+version: "3"
+services:
+  omeka_ecosystem:
+    image: digirati/madoc-platform:latest
+    links:
+      - mysql
+    environment:
+      - APP_ENV=PROD
+      - OMEKA__DATABASE_HOST=omeka_db
+      - OMEKA__DATABASE_NAME=${MYSQL_DATABASE}
+      - OMEKA__DATABASE_USER=${MYSQL_USER}
+      - OMEKA__DATABASE_PASSWORD=${MYSQL_PASSWORD}
+      - OMEKA__DATABASE_PORT=${MYSQL_PORT}
+      - OMEKA__MAIN_SITE_DOMAIN=http://localhost:8888/
```

Comment:
We'll need to hook this up to whatever comes out of the R53 config, maybe as an environment variable

Reply:
I've made this `${MADOC_PLATFORM}` for now so that it can be controlled via environment var.